hci/event/events/vendor_specific/gap_device_information.py

```diff
@@ -53,7 +53,7 @@ def __str__(self):
             GAP_DeviceInformation.EventType(self.event_type).name,
             hex(self.address_type),
             GAP_DeviceInformation.AdressType(self.address_type).name,
-            _bytes_to_hex_string(self.address),
+            _bytes_to_hex_string(self.address, revert=True),
```

Comment:
Properties are responsible for reverting little endian order, not the `__str__` method.